applications/StructuralMechanicsApplication/python_scripts/set_moving_load_process.py

```diff
@@ -0,0 +1,86 @@
+# Importing the Kratos Library
+import KratosMultiphysics
+import KratosMultiphysics.StructuralMechanicsApplication as KSM
+
+def Factory(settings, Model):
+    if not isinstance(settings, KratosMultiphysics.Parameters):
+        raise Exception("expected input shall be a Parameters object, encapsulating a json string")
+    return SetMovingLoadProcess(Model, settings["Parameters"])
+
+from KratosMultiphysics.StructuralMechanicsApplication import set_moving_load_process as smlp
+#
+## All the processes python should be derived from "Process"
+class SetMovingLoadProcess(KratosMultiphysics.Process):
+    """This process assigns given POINT_LOAD and PRESCRIBED_DISPLACEMENT values
+    (scalar) to the DisplacementControl belonging a certain submodelpart.
+    Currently, it is restricted to one node and one global direction.
+
+    Only the member variables listed below should be accessed directly.
+
+    Public member variables:
+    Model -- the container of the different model parts.
+    settings -- Kratos parameters containing solver settings.
+    """
+
+    def __init__(self, Model, settings ):
+        """ The default constructor of the class
+
+        Keyword arguments:
+        self -- It signifies an instance of a class.
+        Model -- the container of the different model parts.
+        settings -- Kratos parameters containing solver settings.
+        """
+
+        KratosMultiphysics.Process.__init__(self)
+
+        #The value can be a double or a string (function)
+        default_settings = KratosMultiphysics.Parameters("""
+        {
+            "help"            : "This process applies a moving load condition belonging to a modelpart. The load moves over line elements.",
+            "model_part_name" : "please_specify_model_part_name",
+            "variable_name"   : "MOVING_LOAD",
+            "load"            : [0.0, 1.0, 0.0],
+            "direction"       : [1,1,1],
+            "velocity"        : 1
```

Comment:
Why not adding the velocity vector instead? If doing it in that way we can eventually add a function depending on x,y,z and t for each component.

Reply:
My idea was to give a velocity table eventualy, such that every time step, the velocity can be different. The direction is meant to be either positive or negative (per axis). But it will always follow the line of the condition. An exact direction is thus dependent on the mesh.

Reply:
but giving a function might be better indeed

Reply:
do you have an idea on how to give such a function as an input?

Reply:
You can check what we do with the value field  in the `AssignVectorVariableProcess`. I'd use a similar mechanism.

Reply:
thanks, i will check it out

Reply:
I was rethinking this comment. I think adding a function for the velocity is great, but in my opinion it should only be based on time, not on x,y,z. Because the moving load is only applied on line elements. In this case, it is still required to keep the "direction" variable, since it is important to identify which direction the load has to move in in the initial solution step. Further directions are not constant, since the load follows the line element. This is handled in the process, since i sort the condition element in the correct order

Reply:
an other thought: maybe in the future a moving load on a surface can be added, where indeed a velocity vector can be set as a function of x,y,z and time. But for now, a moving load on a continuous line has my priority.